tests/widgets_table.rs

```diff
@@ -747,11 +747,94 @@ fn widgets_table_enable_always_highlight_spacing() {
 }
 
 #[test]
-fn widgets_table_can_have_elements_styled_individually() {
+fn widgets_table_can_have_spacing_in_all_columns() {
+    let test_case = |state: &mut TableState, space: HighlightSpacing, expected: Buffer| {
```

Comment:
Use a function instead of a closure and add `#[track_caller]` to make it easier to work out which test fails (the reported failure line will be e.g. line 784 or 800 instead of always line 777)